.travis.yml

```diff
@@ -1,19 +1,107 @@
+sudo: required
+dist: trusty
 language: go
-sudo: false
 go:
   - "1.10.3"
+
 # import forks to the correct location
 go_import_path: github.com/VertebrateResequencing/wr
+
+services:
+  - docker
+
+cache:
+  directories:
+    - $GOPATH/pkg/
+
+env:
+  global:
+    - KUBERNETES_VERSION=v1.10.0
+    # Move Kubernetes config files to the appropriate place and adjust permissions.
+    - CHANGE_MINIKUBE_NONE_USER=true
+    - DEP_VERSION="0.5.0"
+
 before_install:
-  - go get github.com/modocache/gover
-  - go get github.com/mattn/goveralls
-  - go get github.com/Masterminds/glide
-  - $HOME/gopath/bin/glide -q install
-script:
-  - go test -tags netgo -v -coverprofile=queue.coverprofile -covermode count ./queue
-  - go test -tags netgo -v -coverprofile=sched.coverprofile -covermode count ./jobqueue/scheduler
-  - WR_MANAGERPORT="11301" go test -tags netgo -v -coverprofile=jq.coverprofile -covermode count ./jobqueue
-  - go test -tags netgo -v -coverprofile=cloud.coverprofile -covermode count ./cloud
-  - go test -tags netgo -v -coverprofile=rp.coverprofile -covermode count ./rp
-  - $HOME/gopath/bin/gover
-  - $HOME/gopath/bin/goveralls -coverprofile=gover.coverprofile -service travis-ci
+  # - go get github.com/modocache/gover
+  # - go get github.com/mattn/goveralls
+  # - go get github.com/Masterminds/glide
+  # - $HOME/gopath/bin/glide -q install
+  # # wr consists of multiple Go packages, which refer to each other by
+  # # their absolute GitHub path, e.g. github.com/VertebrateResequencing/wr/kubernetes/client.
+  # # That means, by default, if someone forks the repo and makes changes across
+  # # multiple packages within wr, Travis won't pass for the branch on their
+  # # own repo. To fix that, we move the directory
+  # - mkdir -p $TRAVIS_BUILD_DIR $GOPATH/src/github.com/VertebrateResequencing
+  # - test ! -d $GOPATH/src/github.com/VertebrateResequencing/wr && mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/VertebrateResequencing/wr || true
+  # Download the binary to bin folder in $GOPATH
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
+  # Make the binary executable
+  - chmod +x $GOPATH/bin/dep
+
+install:
+  - dep ensure
+
+jobs:
+  include:
+    # - stage: Tests
+    #   script:
+    #     - go test -tags netgo -v -coverprofile=queue.coverprofile -covermode count ./queue
+    #     - go test -tags netgo -v -coverprofile=sched.coverprofile -covermode count ./jobqueue/scheduler
+    #     - WR_MANAGERPORT="11301" go test -tags netgo -v -coverprofile=jq.coverprofile -covermode count ./jobqueue
+    #     - go test -tags netgo -v -coverprofile=cloud.coverprofile -covermode count ./cloud
+    #     - go test -tags netgo -v -coverprofile=rp.coverprofile -covermode count ./rp
+    #     - $HOME/gopath/bin/gover
+    #     - $HOME/gopath/bin/goveralls -coverprofile=gover.coverprofile -service travis-ci
```

Comment:
Probably as the last commit you do at the end of all code review, don't forget to enable the wr tests